src/Microsoft.AspNetCore.WebHooks.Common/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+
+namespace System
+{
+    /// <summary>
+    /// Extension methods for <see cref="string"/>.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public static class StringExtensions
```

Comment:
Move these to a "Internal" namespace. Extension methods on string are generally a bad idea.